conjur/client.py

```diff
@@ -119,7 +119,7 @@ def __init__(self,
                 credentials = CredentialsFromFile(DEFAULT_NETRC_FILE)
                 loaded_netrc = credentials.load(loaded_config['url'])
             except netrc.NetrcParseError as netrc_error:
-                raise Exception("Error: netrc in an invalid format." \
+                raise Exception("Error: netrc in an invalid format. " \
```

Comment:
Sorry to say, but this doesn't look like correct english... ;)
Maybe:
```suggestion
                raise Exception("Error: netrc is in an invalid format. " \
```